### ISSUE TITLE
Local port set as zero if not deinfed on cli

### ISSUE THREAD

**Issue by ondrejj** [nils-ohlmeier/sipsak#71]
After update to version 0.9.8, local port in From/Via/Contact headers is set to 0, which is not compatible with some SIP services. Please, revert to filling local port even if it's not defined on command line. Example request from sipsak 0.9.8:

```
OPTIONS sip:8111@158.197.100.9 SIP/2.0
Via: SIP/2.0/UDP 158.197.160.75:0;branch=z9hG4bK.5ccef639;rport;alias
From: sip:sipsak@158.197.160.75:0;tag=1c2eedec
To: sip:8111@158.197.100.9
Call-ID: 472837612@158.197.160.75
CSeq: 1 OPTIONS
Contact: sip:sipsak@158.197.160.75:0
Content-Length: 0
Max-Forwards: 70
User-Agent: sipsak 0.9.8
Accept: text/plain
```

Request from sipsak 0.9.7:

```
OPTIONS sip:8111@158.197.100.9 SIP/2.0
Via: SIP/2.0/UDP 158.197.160.75:38865;branch=z9hG4bK.05d91154;rport;alias
From: sip:sipsak@158.197.160.75:38865;tag=290e7bcc
To: sip:8111@158.197.100.9
Call-ID: 688815052@158.197.160.75
CSeq: 1 OPTIONS
Contact: sip:sipsak@158.197.160.75:38865
Content-Length: 0
Max-Forwards: 70
User-Agent: sipsak 0.9.7
Accept: text/plain
```

Using --local-port parameter can workaround this, but it can make more trouble when running in parallel.

**Comment by nils-ohlmeier** [comment]
Thank you for the report.

This is exactly the kind of problems I was concerned about, when I did the big refactoring for gcc-10. I'll have a look at where I introduced that bug.

**Comment by nils-ohlmeier** [comment]
The issue should be fixed in release 0.9.8.1.

Thanks again for the report!
